web/package.json

```diff
@@ -7,7 +7,7 @@
     "scaffold": "mkdir -p dist && cp src/index.html dist/",
     "build": "npm run scaffold && NODE_ENV=production webpack -p",
     "lint": "eslint 'src/**/*.js'",
-    "start": "webpack-dev-server --host 0.0.0.0 --port 8000 --history-api-fallback --content-base src/",
+    "start": "webpack-dev-server --host 0.0.0.0 --port 8001 --history-api-fallback --content-base src/",
```

Comment:
Ha.  I can see why having both the API and webpack-dev-server running on port 8000 might be a problem.  😛